### ISSUE TITLE
Term request: surveillance process

### ISSUE THREAD

**Issue by PhiBabs935** [obi-ontology/obi#1181]
_surveillance process_ =def Planned process with the objective to produce information about some evaluant with the purpose of, if justified by the information gathered, managing, directing, or protecting 

is an OBI:_planned process_ that was originally created for the Vector Surveillance Management Ontology (VSMO) to serve as parent for terms such as _pathogen surveillance_ and _vector surveillance_ 

Since VSMO hasn't been actively maintained in some time (not sure if it was officially obsoleted), some of us in the Infectious Disease Ontology team are planning to rebrand the latter two terms as IDO terms. 

But since _surveillance process_ didn't seem appropriate for inclusion in IDO, we suggested that it be added to OBI instead. [see the discussion here https://osf.io/az6u5/, specifically the bottom of p. 17]

I am also submitting this term request because Chris Stoeckert informed me that his team needed _surveillance process_ for their ClinEpiDB project (https://clinepidb.org/ce/app/). Chris wants to bring this issue up at tomorrow's OBI call.

**Update July 26** 
The new suggestion is to define "surveillance process" as a child of OBI's _investigation_

_Surveillance process_: =def. An investigation in which some entity is monitored, and data concerning that entity collected, interpreted, and disseminated, for the purpose of managing, directing, or protecting that entity, or some other entity.

Notes: I used ‘entity’ because I wanted to keep it general enough to include surveillance of material entities (e.g. people, pathogens) processes (their activities or behaviors), sites (towns, countries), and information content entities (e.g. surveillance of my phone’s data) etc.  
-Concerning "protecting that entity, or some other entity" I might be the entity under surveillance in a case where it is intended for my own protection, or I might be under surveillance for the purpose of protecting another person (e.g. someone else that I pose a potential threat to).

**Comment by cstoeckert** [comment]
Discussed on June 22, 2020 call. Need to expand on definition as it seems truncated.
Also need to consider is this being defined as an assay with evaluants as input and information as ouput.

**Comment by PhiBabs935** [comment]
Although, now that I think of it, the suggested definition may need slight rewording to handle the case where the entity under surveillance is distinct from the entity protected, e.g. when a vector population is under surveillance in order to assist in management of the vector, and thereby protect a human population threatened by the spread of the relevant pathogens.

**Comment by cstoeckert** [comment]
I thought that was covered with the ‘or’s in “for the purpose of managing, directing, or protecting that entity, or some other entity.”
Chris

**Comment by PhiBabs935** [comment]
I guess my worry is that someone could be read this as:
for the purpose of: managing, directing, or protecting that entity OR managing, directing, or protecting some other entity. 
where the OR is read exclusively

**Comment by PhiBabs935** [comment]
Hmm. Not sure why I thought it was a problem. The definition looks fine to me in hindsight.

**Comment by cstoeckert** [comment]
Great. I will bring the July 26 updated definition (Surveillance process: =def. An investigation in which some entity is monitored, and data concerning that entity collected, interpreted, and disseminated, for the purpose of managing, directing, or protecting that entity, or some other entity.) to an upcoming OBI call.

**Comment by nandadorea** [comment]
Hi, Damion Dooley brought this discussion to my attention (thanks!). I have been working to try to bring the Health Surveillance Ontology (https://bioportal.bioontology.org/ontologies/HSO) up to OBO standards, hope to join the family soon. I agree in full that OBI represents the best starting point for a surveillance process. in HSO, all concepts are centered around a process called "surveillance activity". I went back and forth a lot about whether to place it as a subclass of OBI's investigation, but ultimately decided not to because I was unsure that we could always fulfill the "study design" component. Some surveillance activities are surveys planned with a specific study design. Some are just a reaction to an outbreak, and although there is a plan specification (making it still a planned process), I wasn't sure about a study design. My ambition was to soon be able to specify better the different types of surveillance activities (or processes) and some of them would be an investigation, but some may not be, I think (but happy to discuss).

I agree that it can only be good to go ahead and add the surveillance process as defined above to OBI, which will help a lot of other domains, as Shane pointed out. The only question I would bring to discussion is **whether it should be an investigation, or its own type of planned process.** In HSO we modelled "surveillance activity" as a planned process and added surveillance relevant classes under "plan specification", "action specification" and "objective specification".

Wherever the "surveillance process" ends up in OBI, I will make sure to map to it in HSO, and follow the agreements. If it is a general type of planned process, I may be able to use the OBI term directly as my root (replacing what I had called a "surveillance activity"). If it is placed under investigation, I think I should still be able to map the details of an activity that make some of them be the same as a surveillance process in OBI, and some others potentially not. The goal of an ontology focused specifically in surveillance is exactly to detangle these differences, so I still think it is OK. 

Just as a p.s., there is a lot of terminology disagreements about surveillance terms between animal health, human health and food safety people. By now I have learned a lot which terms to avoid, so that we are not stuck in words, and can get through to discussing meaning. "surveillance activity" and "surveillance process" are both "safe terms" in my opinion.
